analytics/terraform/datahub-on-eks/datahub-eks/values/datahub_values.yaml

```diff
@@ -0,0 +1,334 @@
+# Values to start up datahub after starting up the datahub-prerequisites chart with "prerequisites" release name
+# Copy this chart and change configuration as needed.
+datahub-gms:
+  enabled: true
+  image:
+    repository: linkedin/datahub-gms
+    tag: "v0.8.45"
+
+datahub-frontend:
+  enabled: true
+  image:
+    repository: linkedin/datahub-frontend-react
+    tag: "v0.8.45"
+  # Set up ingress to expose react front-end
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: alb
+      alb.ingress.kubernetes.io/scheme: internet-facing
```

Comment:
We need a note to users wherever we use internet-facing load balancers saying that this is only for demo purpose. Users should change this when they run in prod

Reply:
done - added comment in the doc along with the value section that needs to be changed